### ISSUE TITLE
Gear vr support

### ISSUE THREAD

**Issue by andreasplesch** [x3dom/x3dom#635]
There is an official internet browser in the Oculus app store which supports webvr via a setting:

http://developer.samsung.com/technical-doc/view.do?v=T000000270L

It may be possible to get x3dom working on this browser for Gear VR support.

**Comment by mkeblx** [comment]
Yes, WebVR on the Gear is using the 1.0 API draft so just getting it to work with the 1.0 API should be sufficient, and will then work on Rift, Vive. I'm working on getting some x3dom samples to run on Gear so will share results.
